src/lib/cli/command.js

```diff
@@ -498,7 +498,7 @@ args.argv = async function ( argv, cb ) {
 		res = await cb( this.sub, options );
 		if ( _opts.format && res ) {
 			if ( res.header ) {
-				console.log( formatData( res.header, 'keyValue' ) );
+				if ( options.format !== 'json' ) console.log( formatData( res.header, 'keyValue' ) );
```

Comment:
```suggestion
				if ( options.format !== 'json' ) {
				 console.log( formatData( res.header, 'keyValue' ) );
				}
```

with proper tab spacing to maintain syle